File: EpersBackend/Epers.Models/Nomenclatoare/SetareProfil.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Epers.Models.Nomenclatoare
{
	public class SetareProfil
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public int? Id_Skill { get; set; }
        public int? Id_Post { get; set; }
        public int? Ideal { get; set; }
    }
}

File: epers-front-end/src/app/models/nomenclatoare/SetareProfil.ts
export interface SetareProfil {
    id: number;
    id_Skill: number | null;
    id_Post: number | null;
    ideal: number | null;
}

export interface TableSetareProfil {
    setareProfil: SetareProfil;
    denumireSkill: string;
    descriereSkill: string;
    selected: boolean;
}

